mlflow/utils/autologging_utils.py

```diff
@@ -426,6 +426,20 @@ def __call__(self, original, *args, **kwargs):
                 raise e
 
 
+class _AutologgingSessionManager:
+    _session = False
+
+    @classmethod
+    def start_session(cls):
+        cls._session = True
```

Comment:
Rather than `True` / `False`, can we keep track of the autologging integration corresponding to the session. E.g., `_session` becomes `None` or `mlflow.sklearn` / `mlflow.tensorflow` / etc